dhis-2/dhis-api/src/main/java/org/hisp/dhis/dashboard/Dashboard.java

```diff
@@ -52,6 +54,16 @@
 
     private List<DashboardItem> items = new ArrayList<>();
 
+    /**
+     * Whether we restrict the filter dimensions the user may select.
+     */
+    private Boolean filterDimensionsAreRestricted = false;
```

Comment:
Better to use primitive boolean, as we then can have just two options (true or false) instead of three (true, false or null).